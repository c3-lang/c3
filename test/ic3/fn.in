quote fn (x) { x }
quote fn (x, _y) { x }
quote fn ([x | _y]) { x }
quote fn {
  ([]) { :error }
  ([x | _y]) { x }
  (_) { :error }
}
fn (x) { x }
fn (x, _y) { x }
fn ([x | _y]) { x }
fn {
  ([]) { :error }
  ([x | _y]) { x }
  (_) { :error }
}
