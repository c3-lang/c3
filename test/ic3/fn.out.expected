quote fn (x) { x }

quote fn (x, _y) { x }

quote fn ([x | _y]) { x }

fn (x) { x }

fn (x, _y) { x }

fn ([x | _y]) { x }

